src/components/Main/Main.js

```diff
@@ -1,24 +1,20 @@
 import React from 'react';
-import FavButton from '../FavButton/FavButton';
-import Button from '../Button/Button';
-import DataView from '../DataView/DataView';
 import './Main.css';
+import FavButton from '../FavButton/FavButton.js';
+import Button from '../Button/Button.js';
+import DataView from '../DataView/DataView.js';
 
-const Main = ({ favorites }) => {
-  return (
-    <div className='main-container'>
-      <div className='title'>
-        <p>SWAPI-Box</p>
-        <FavButton favorites={favorites}/>
-      </div>
-      <div className='buttons-container'>
-        <Button />
-        <Button />
-        <Button />
-      </div>
+const Main = ({favorites})=>{
+  return(
+    <div>
```

Comment:
This change breaks tests because the className is required for the tests. Also, the removal of the semi-colons kicks back ESLint errors.